rs/bitcoin/ckbtc/minter/src/lib.rs

```diff
@@ -189,13 +189,13 @@ fn compute_min_withdrawal_amount(
     const PER_REQUEST_RBF_BOUND: u64 = 22_100;
     const PER_REQUEST_VSIZE_BOUND: u64 = 221;
     const PER_REQUEST_MINTER_FEE_BOUND: u64 = 305;
-    const PER_REQUEST_KYT_FEE: u64 = 2_000;
+    const PER_REQUEST_CHECK_FEE: u64 = 2_000;
```

Comment:
The fee should be 100!

Reply:
I actually don't understand why this is a constant. It should come from `CkBtcMinterState::kyt_fee`, no?

Reply:
Yeah, good question. Can we get rid of this constant?